### ISSUE TITLE
Feature request: Unstarred Clips removal

### ISSUE THREAD

**Issue by anjanoleti** [clipto-pro/Desktop#32]
Please add an easy way to select and/or delete unstarred notes.

**Comment by atrashler** [comment]
> Please add an easy way to select and/or delete unstarred notes.

Now it is possible to select multiple notes and delete the selection with 'Delete' action.
In your request you mean that in such selection 'Starred' notes can be presented and it should be a way to exclude it from the selection, isn't it?

**Comment by anjanoleti** [comment]
Yes. Looking for a way to quickly select just the unstarred notes.

**Comment by gth7754125** [comment]
need it too!!  Maybe add an timer to auto clear these temp and unnecessary msg is a good way.

**Comment by atrashler** [comment]
@gth7754125 , @anjanoleti , I have released the new version 2.8.1 with ability to configure the limit of notes from clipboard. As I understand, this issue relates to the functionality 'Notes from Clipboard', isn't it?

**Comment by atrashler** [comment]
> Yes. Looking for a way to quickly select just the unstarred notes.

Does it relate to the functionality 'Notes from Clipboard' or for any notes? If only for 'Notes from Clipboard' then now it is possible to 'Clean' this category from all untagged and unstarred notes. Also now it is possible to configure its limit to clean it automatically.

**Comment by gth7754125** [comment]
Just for 'Notes from clipboard' is OK.  
because some people may use password manager such as 1password, so the clipboard records on Clipto need to be cleared regularly. Although i can set it on the password manager app, but sometimes it makes inconvenient.

**Comment by atrashler** [comment]
> Just for 'Notes from clipboard' is OK.
> because some people may use password manager such as 1password, so the clipboard records on Clipto need to be cleared regularly. Although i can set it on the password manager app, but sometimes it makes inconvenient.

idea for the feature: for ‘Notes from Clipboard’ add ability to configure “exclude rules” to skip saving of a clip if it matches such rules. The only problem is how to make this rules to be simple and useful (regexp, equality, something else...)

**Comment by gth7754125** [comment]
simple is the best.
just add a timer to clear the record from clipboard regularly is fine.
